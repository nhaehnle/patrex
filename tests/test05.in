{
	a.foo(get(1, x), "hi", tmp, 0);
	b->foo(y, "bar", tmp, 0);
}
