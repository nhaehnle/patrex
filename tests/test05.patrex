define arg $!(,)*

match $( ${id} $|( . )( -> ) )|var| foo ( ${arg}, ${arg}|second|, ${arg}|third|, ${arg} ); $<|pos|
	erase second.end third.end
	insert pos "\n\t{var}bar({third});"
