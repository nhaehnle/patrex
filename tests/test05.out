{
	a.foo(get(1, x), "hi", 0);
	a.bar(tmp);
	b->foo(y, "bar", 0);
	b->bar(tmp);
}
